cid-redirects.json

```
@@ -1125,9 +1125,10 @@
   "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Imperva_-_Incapsula_Web_Application_Firewall": "/docs/integrations/security-threat-detection/imperva-incapsula",
   "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Imperva_-_Incapsula_Web_Application_Firewall/01Collect-logs-for-the-Imperva-Incapsula-Web-Application-Firewall-App": "/docs/integrations/security-threat-detection/imperva-incapsula",
   "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Imperva_-_Incapsula_Web_Application_Firewall/Install-the-Incapsula-App-and-view-the-Dashboards": "/docs/integrations/security-threat-detection/imperva-incapsula",
-  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security": "/docs/integrations/security-threat-detection/keeper-security",
-  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security/Collect_Logs_for_Keeper_Security": "/docs/integrations/security-threat-detection/keeper-security",
-  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security/Install_the_Keeper_Security_App_and_view_the_Dashboards": "/docs/integrations/security-threat-detection/keeper-security",
+  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security": "/docs/integrations/partner-ecosystem-apps",
+  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security/Collect_Logs_for_Keeper_Security": "/docs/integrations/partner-ecosystem-apps/",
+  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security/Install_the_Keeper_Security_App_and_view_the_Dashboards": "/docs/integrations/partner-ecosystem-apps/",
+  "/docs/integrations/security-threat-detection/keeper-security": "/docs/integrations/partner-ecosystem-apps/",
```

Docusaurus can sometimes be weird about trailing slashes in URLs

```suggestion
  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security/Collect_Logs_for_Keeper_Security": "/docs/integrations/partner-ecosystem-apps",
  "/07Sumo-Logic-Apps/22Security_and_Threat_Detection/Keeper_Security/Install_the_Keeper_Security_App_and_view_the_Dashboards": "/docs/integrations/partner-ecosystem-apps",
  "/docs/integrations/security-threat-detection/keeper-security": "/docs/integrations/partner-ecosystem-apps",
```